docs/pages/lighting-setups.rst

```diff
@@ -0,0 +1,197 @@
+Working with Lights
+###################
+
+:summary: This tutorial demonstrates the creation and manipulation of LightSetups in Habitat-sim.
+
+.. contents::
+    :class: m-block m-default
+
+A **LightSetup** consists of a set of LightInfo structures defining the common configuration
+of a set of point lights used to render objects in a scene. Once defined and registered,
+a LightSetup can be assigned to any subset of objects in the scene, including the scene asset itself.
+
+Each **LightInfo** structure in a LightSetup defines the `color`, `position`, and `LightPositionModel` of a single point light source.
```

Comment:
```suggestion
Each **LightInfo** structure in a LightSetup defines the ``color``, ``position``, and ``LightPositionModel`` of a single point light source.
```

reST needs two backticks for inline code. Here it might have accidentally done the correct thing, but only because it interpreted those as `:ref:`s (which is the default here), failed to find the target and rendered as monospace:

```
WARNING:root:Sphinx symbol `color` not found, rendering as monospace
WARNING:root:Sphinx symbol `position` not found, rendering as monospace
WARNING:root:Sphinx symbol `LightPositionModel` not found, rendering as monospace
```